executor/runcexecutor/executor_common.go

```diff
@@ -1,88 +1 @@
-//go:build !linux
```

Comment:
@profnandaa When you are adding runc wcow support you likely need to add this file back, but atm I didn't find any way this file can be called. Only importer from `cmd/oci_worker` already had linux-only build tags.

Reply:
Noted, thanks for letting me know.